src/models/Component.js

```diff
@@ -156,6 +156,31 @@ class Component extends FileInformation {
       });
   }
 
+  /**
+   * Remove id in link attribute corresponding to the given name.
+   * @param {String} name - Name of attribute to remove
+   * @param {String} id - Id to remove.
+   */
+  removeLinkAttribute(name, id) {
+    this.attributes = this.attributes
+      .filter((attribute) => {
+        if (attribute.name !== name) {
+          return true;
```

Comment:
Shouldn't Component.getAttributeByName (or __getAttributeByName) be used here instead?

Reply:
no because we want to filter the attribute list without the attribute if its value is `[]`